datastore/h2gis/pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>gt-jdbc</artifactId>
             <version>${geotools.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-referencing</artifactId>
+            <version>${geotools.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+            <version>${geotools.version}</version>
+        </dependency>
```

Comment:
If those dependencies are used for tests cases they should have the tag `<scope>test</scope>`